.circleci/config.yml

```diff
@@ -900,23 +900,25 @@ jobs:
       MAKEFLAGS: -j10
     steps:
       - checkout
-      - restore_cache:
-          keys:
-            - dependencies-osx-{{ arch }}-{{ checksum ".circleci/osx_install_dependencies.sh" }}
-      # DO NOT EDIT between here and save_cache, but rather edit ./circleci/osx_install_dependencies.sh
-      # WARNING! If you do edit anything here instead, remember to invalidate the cache manually.
+      # FIXME: Dependency caching disabled on macOS because it takes longer than just installing them each time.
+      # See https://github.com/ethereum/solidity/issues/12925
+      #- restore_cache:
+      #    keys:
+      #      - dependencies-osx-{{ arch }}-{{ checksum ".circleci/osx_install_dependencies.sh" }}
+      ## DO NOT EDIT between here and save_cache, but rather edit ./circleci/osx_install_dependencies.sh
+      ## WARNING! If you do edit anything here instead, remember to invalidate the cache manually.
```

Comment:
Is it better to leave it commented out like this so that it's easy to restore or should I just delete it?

Reply:
It's nothing really special - and we still have it in the git history... I'd just remove it, but also fine with a comment.

Reply:
ok. Removed commented-out code and left only the FIXME.